packages/bsky/src/hydration/hydrator.ts

```diff
@@ -650,6 +651,19 @@ export class Hydrator {
     const listMemberDids = listsMembers.flatMap((lm) =>
       lm.listitems.map((li) => li.did),
     )
+    const listBlocks: Map<string, Blocks> = new Map()
+    await Promise.all(
+      listUris.map(async (uri) => {
```

Comment:
I'm not sure that breaking the blocks up by list helps us out a ton, can probably get rid of that idea.
```ts
const listCreatorMemberPairs = [...listMembersByList.entries()]
  .flatMap(([listUri, members]) => {
    const creator = didFromUri(listUri)
    return members.map((member) => [creator, member.did])
  })

const blocks = await this.graph.getBidirectionalBlocks(listCreatorMemberPairs)

// ...

const creator = didFromUri(sp.record.list)
// update aggregation with list items for top 12 most followed members
agg.listItemSampleUris = [...members.listitems]
  .filter((li) => !blocks.isBlocked(creator, li.did))
  // ...
```